contracts/test/Token.sol

```diff
@@ -23,20 +24,73 @@ contract Token is ERC20 {
         require(!_blocked[to], "Token transfer refused. Receiver is on blacklist");
         super._beforeTokenTransfer(from, to, amount);
     }
+}
+
+contract TokenNoReturn {
```

Comment:
This is required to get full coverage on erc20_safe_transfer and erc20_safe_transferFrom